docs/examples/droplet_fusion/droplet_fusion.rst

```diff
@@ -0,0 +1,200 @@
+
+Droplet fusion
+==============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+.. _droplet_fusion:
+
+Analyzing a droplet fusion event
+--------------------------------
+
+Phase separated protein droplet can fuse. The relaxation time of the fusion, together with the size of the droplets can say something about the material properties of the droplet.
+In the data analyzed in this Notebook, two droplets that are brought together at a constant speed using optical tweezers. 
+We will obtain the relaxation time from the force signal and estimate the size of the droplets from the scan.
+
+Download the droplet fusion data
+--------------------------------
+
+The droplet fusion data are stored on zenodo.org.
+We can download the data directly from Zenodo using the function :func:`~lumicks.pylake.download_from_doi`.
+The data will be stored in the folder called `"test_data"`::
+
+    filenames = lk.download_from_doi("10.5281/zenodo.12772709", "test_data")
+
+Relaxation time of fusion event
+-------------------------------
+
+First, plot the force signal using Pylake. The force has a characteristic jump during a fusion event. 
+Since the force calibration breaks down during the fusion event, the absolute value of the force is not reliable, and we label the y-axis as 'laser signal'::
+
+    plt.figure()
+    f["Force HF"]["Force 2x"].plot()  
+    plt.ylabel("Laser signal (a.u.)")
+    plt.show()
+
+.. image:: force_signal.png
+
+Select data for fit
+^^^^^^^^^^^^^^^^^^^
+Below we are selecting the force and trap data at the fusion event. When fitting the fusion relaxation time, 
+it is important that the traps holding the droplets are either both static, or one of the traps is moving at a constant speed. 
+Therefore, we plot the trap position over time::
+
+    start = "5.4s"
+    stop = "5.9s"
+
+    force_selection = f.force2x[start:stop]
+    trap_selection = f["Trap position"]["1X"][start:stop]
+
+    plt.figure()
+    plt.subplot(2, 1, 1)
+    force_selection.plot()
+    plt.ylabel("Laser signal (a.u.)")
+    plt.subplot(2, 1, 2)
+    trap_selection.plot()
+    plt.ylabel("x-coordinate ($\mu$m)")
+    plt.tight_layout()
+    plt.show()
+
+.. image:: selected_data.png
+
+Model for fusion
+^^^^^^^^^^^^^^^^
+
+We will be fitting the force data during the fusion event with the following equation: $f(t) = ae^{-t/\tau}+bt+c$                 
+
+The term $bt$ accounts for the movement of the trap, assuming a constant trap speed [1]_. The parameter of interest is $\tau$, the relaxation time schale of the fusion event::
+
+    from scipy.optimize import curve_fit
+
+    def relaxation_model(t, tau, a,b,c):
+        return a*np.exp(-t / tau)+b*t+c
+
+Fit the data and plot the result::
+
+    time = force_selection.seconds
+    force = force_selection.data
+
+    popt, pcov = curve_fit(relaxation_model, time, force, [0.1,force[0],0,0])
+    print(popt)
```

Comment:
Minor, but it might be nice to format this a little bit.

Something like (I didn't test this):
```suggestion
    tau, amplitude, movement_term, offset = popt
    print(f"Relaxation time: {tau:.2f}, amplitude: {amplitude:.2f}, movement_term: {movement_term:.2f}, offset: {offset:.2f}")
```